### DISCUSSION TITLE
Notebook, VSCode, and Copilot Compatibility

### DISCUSSION THREAD

**Discussion by xandre-clementsmith** [community/community#121871] [Copilot Conversations]
I am attempting to use github copilot within a VSCode jupyter notebook, but receive an error whenever it decides to run /fix:

"Illegal argument: InlineFix3Prompt should not be used with a notebook!"

I haven't found this error anywhere else, so want to determine whether this is a bug, or if copilot cannot support notebooks for this specific function. Most copilot functionality is working fine in a VSCode notebook, but I can't prohibit it from trying to use /fix if it interprets that as the best approach. Believe there's a similar issue that has been reported with [PyCharm/DataSpell](https://github.com/orgs/community/discussions/17185), but may have been resolved.

Windows OS. Python3. Happy to provide any additional information. 

![image](https://github.com/community/community/assets/126746075/ee66f245-7d58-4ef0-99c7-b40a9cb21c37)

**Comment by jsboige** [comment]
I'm having the same issue with .Net interactive notebooks in polyglot extension
